src/preferences/PreferencesManager.js

```diff
@@ -528,7 +532,7 @@ define(function (require, exports, module) {
      * @return {boolean} true if a value was set
      */
     function setValueAndSave(id, value, options) {
-        DeprecationWarning.deprecationWarning("setValueAndSave called for " + id + ". Use set instead.");
+        DeprecationWarning.deprecationWarning("PreferencesManager.setValueAndSave() called for " + id + ". Use set instead.", true);
```

Comment:
To be consistent, we should change `set` to `PreferencesManager.set()`, too.

Otherwise, this looks good and we definitely want this in Release 1.4.